doc/DevTeam.md

```diff
@@ -0,0 +1,41 @@
+# Development Team
+
+## Core Members
+Mentors: Damith C. Rajapakse, Xie Kai
+
+Team Leads: Ian Teo, Ong Heng Le
+
+Senior Devs: Ang Shi Ya, Jamos Tay, Kenneth Loh, Syed Abdullah
+
+## Project Areas
+Area | Responsibilities | Lead | Co-Lead | Past Leads
```

Comment:
I wonder if we should change this to our names.

Reply:
not sure, but I was thinking to include this after we become senior devs